config.toml

```diff
@@ -158,6 +158,10 @@ address_type = { derivation = 'cosmos' }
 # Recommended value for Cosmos SDK: 'ibc'
 store_prefix = 'ibc'
 
+# Specify the folder used to store the keys. Optional
+# If this is not specified then the hermes home folder is used.
+# key_store_folder = '/home/.hermes_keys'
```

Comment:
Let's put this right under `key_name`? Otherwise it's a bit confusing as the store prefix is unrelated to the key store.